app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,66 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/background"
     tools:context=".MainActivity">
 
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/activity_main_appIcon"
+        android:layout_width="@dimen/app_icon_width"
+        android:layout_height="@dimen/app_icon_height"
+        android:src="@drawable/icon"
+        app:layout_constraintBottom_toTopOf="@+id/activity_main_enter_city_text_view"
+        app:layout_constraintEnd_toStartOf="@+id/activity_main_app_name_text_view"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
     <TextView
+        android:id="@+id/activity_main_app_name_text_view"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        android:text="@string/activity_main_app_name_text"
+        android:textColor="@color/app_name_text_color"
+        android:textSize="@dimen/app_name_text_size"
+        app:layout_constraintBottom_toTopOf="@+id/activity_main_enter_city_text_view"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/activity_main_appIcon"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:id="@+id/activity_main_enter_city_text_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/activity_main_enter_city_name_text"
+        android:textAlignment="center"
+        android:textColor="@color/enter_city_text_color"
+        android:textSize="@dimen/enter_city_text_size"
```

Comment:
Here you can move some properties to a style

Reply:
done